.github/workflows/StealthyStash.yml

```
@@ -3,147 +3,279 @@ on:
+    container: ${{ matrix.swift.nightly && format('swiftlang/swift:nightly-{0}-{1}', matrix.swift.version, matrix.os) || format('swift:{0}-{1}', matrix.swift.version, matrix.os) }}
+    if: ${{ !contains(github.event.head_commit.message, 'ci skip') }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [noble, jammy]
+        swift:
+          - version: "5.9"
+          - version: "5.10"
+          - version: "6.0"
+          - version: "6.1"
+          - version: "6.1"
```

actionlint (1.7.7)

duplicate value {"nightly": "true", "version": "6.1"} is found in matrix "swift". the same value is at line:22,col:13

(matrix)